cijoe/scripts/bench_reporter.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+"""
+Produce a performance report
+============================
+
+Create it for the current results or overwrite via step-args.
+
+Step Args
+---------
+
+step.with.xnvme_source:  path to xNVMe source (default: config.options.repository.path)
+
+Retargetable: True
+------------------
+"""
+import errno
+from pathlib import Path
+from shutil import copyfile
+
+
+def main(args, cijoe, step):
+    """Primary entry-point"""
+
+    templates_path = Path(
+        step.get("with", {}).get("templates", Path.cwd() / "templates" / "perf_report")
+    ).resolve()
+    # results_path = Path(
+    #    step.get("with", {}).get("results", cijoe.output_path)
+    # ).resolve()
+    # artifacts_path = results_path / "artifacts"
```

Comment:
Is there a need to keep this commented code?

Reply:
nope, removing it